src/app/js/public/resource/Detail.js

```diff
@@ -116,8 +116,8 @@ export const DetailComponent = ({
     sharingUri,
     backToListLabel,
 }) => {
-    const topFields = fields.slice(0, 2);
-    const otherFields = fields.slice(2);
+    const topFields = fields.slice(0, Number(topFieldsCount));
```

Comment:
And what about `config.json` where `topFieldsCount` is not present?
We should add a default value.